2-mandatory/3-shopping-list.js

```diff
@@ -19,7 +19,17 @@ let pantry = {
 };
 
 function createShoppingList(recipe) {
-  // write code here
```

Comment:
This looks good - but one small suggestion: If you are already using an array method here (`forEach`) - maybe the `filter` method would be a better fit for what you are doing here.
It will be good practice to re-write this using `filter` if you get a chance.